test/e2e/framework/util.go

```diff
@@ -786,6 +786,27 @@ func WaitForPersistentVolumeClaimsPhase(phase v1.PersistentVolumeClaimPhase, c c
 	return fmt.Errorf("PersistentVolumeClaims %v not all in phase %s within %v", pvcNames, phase, timeout)
 }
 
+// WaitTimeoutForEvent waits the given timeout duration for an event to occur.
+func WaitTimeoutForEvent(c clientset.Interface, namespace, eventSelector, msg string, timeout time.Duration) error {
```

Comment:
i don't think we should be moving functions back into util.go, client based utilities should live in sub-packages and famework/wait is a good one. there is ongoing work to potentially only have essentials in util.go.

Reply:
if you think the utility is not fine to be in framework/pod, can it be in a different sub-package?

Reply:
@neolit123 

Thank you for your comment. 
Are you suggesting to move it from `framework/pod` to something like `framework/event`?

This PR is trying to use it for a volume event, not for a pod event.
(Actually, we can use it as is, but the name is confusing, so I'm renaming and moving it.)

Reply:
`framework/event` makes sense to me.
it can hold `v1.Event*` related utils.

we seem to have some of these under util.go already.

/cc @oomichi 
WDYT?

Reply:
Moved to `common/events.go`
Adding sig node for review this part. 

/sig node

@Random-Liu @dchen1107 Could you review this PR?

Reply:
I think current PR way(`common/events.go`) is good for me, because the function is used for the test only.

Reply:
ok, moving the utility out of the framework SGTM too.